libs/blocks/global-navigation/global-navigation.js

```diff
@@ -86,7 +86,7 @@ const decorateSignIn = async ({ rawElem, decoratedElem }) => {
         signIn();
       });
     } else {
-      lanaLog({ message: 'Sign in link not found in dropdown.' });
+      lanaLog({ message: 'Sign in link not found in dropdown.', tags: ['error', 'global-navigation'] });
```

Comment:
Maybe `gnav` is better suited to be consistent with other tags. Also, I would make this a `warn` since it's related to authoring + it does not break the navigation.